### PR TITLE
feat(web): use tailwinds font size 

### PR DESCRIPTION
**Pull request by silkeholmebonnen**
## Issue

[AVO-298](https://linear.app/electricitymaps/issue/AVO-298/fix-font-size-issues)

## Description

The goal of this PR is to use tailwinds font size instead of defining our own. This PR also adds styling for h1, h2, h3 and p, and changes the app sizes in the app to reflect the design. 

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

**Comment by silkeholmebonnen** [comment]
This PR is a work in progress for me and @Alportan to collaborate.

**Comment by madsnedergaard** [comment]
Ah wait, now I get why it wasn't a draft - sorry!

**Comment by Alportan** [comment]
Looks good SIlke! I think it definitely solves quite some accessibility issues with the very small font sizes we have today 🎉

I added a few points[ in Figma ](https://www.figma.com/design/O1DtS6UyZHSH4IymG8Ufsw/%F0%9F%8E%B9-Design-Hack?node-id=4452-2279&t=VZ0zMY6GtiezWgip-1)on some screenshots I took from this deployment. Let me know in case you have any follow-up questions. Just leave a comment directly in Figma or here 🙌

Can't wait to release this update 🤩 🚀

**Comment by silkeholmebonnen** [comment]
I have made some changes based on your comments in figma:) Will you have a look again @Alportan ?

**Comment by Alportan** [comment]
> I have made some changes based on your comments in figma:) Will you have a look again @Alportan ?

Looking good!  🙌 Just last thing I noticed was the segmented control, it works as intended on mobile and smaller resolutions under 1279px width, but for 1280px and above, the control shrinks. I've attached a screenshot for reference:
 
image

**Comment by VIKTORVAV99** [comment]
> > I have made some changes based on your comments in figma:) Will you have a look again @Alportan ?
> 
> Looking good! 🙌 Just last thing I noticed was the segmented control, it works as intended on mobile and smaller resolutions under 1279px width, but for 1280px and above, the control shrinks. I've attached a screenshot for reference:
> 
> image

This arrow seem to have grown bigger as well, which is kind of nice but there is a lot of padding around it as well.
image

**Comment by Alportan** [comment]
> > > I have made some changes based on your comments in figma:) Will you have a look again @Alportan ?
> > 
> > 
> > Looking good! 🙌 Just last thing I noticed was the segmented control, it works as intended on mobile and smaller resolutions under 1279px width, but for 1280px and above, the control shrinks. I've attached a screenshot for reference:
> > image
> 
> This arrow seem to have grown bigger as well, which is kind of nice but there is a lot of padding around it as well. image

Good catch, maybe it's a tad too big 😅 Let's keep it as it was before for now 🙌

**Comment by silkeholmebonnen** [comment]
> > I have made some changes based on your comments in figma:) Will you have a look again @Alportan ?
> 
> Looking good! 🙌 Just last thing I noticed was the segmented control, it works as intended on mobile and smaller resolutions under 1279px width, but for 1280px and above, the control shrinks. I've attached a screenshot for reference:
> 
> image

I have adjusted it to always have full width. Was that what you wantedf?

There is still one issue though, on ipad mini it is too wide so it doesn't look very good. I don't know what the best way to solve this is? Should we change it to only say Electricity? That would fix it i guess

image

**Comment by Alportan** [comment]
> > > I have made some changes based on your comments in figma:) Will you have a look again @Alportan ?
> > 
> > 
> > Looking good! 🙌 Just last thing I noticed was the segmented control, it works as intended on mobile and smaller resolutions under 1279px width, but for 1280px and above, the control shrinks. I've attached a screenshot for reference:
> > image
> 
> I have adjusted it to always have full width. Was that what you wantedf?
> 

Yes, exactly! 🥳 

> There is still one issue though, on ipad mini it is too wide so it doesn't look very good. I don't know what the best way to solve this is? Should we change it to only say Electricity? That would fix it i guess
> 
> image

Hmm, indeed! I think we can do that in a PR coming after this one, but even though it looks odd, only 1.9% of users use the tablet version according to Plausible, so not sure it will have a big impact. Of course, we should correct it, but it should be fine!

**Comment by silkeholmebonnen** [comment]
Okay I think it is ready to be reviewed now then!

File: web/src/features/charts/ChartTitle.tsx [hunk 1]
```diff
@@ -21,9 +21,9 @@ export function ChartTitle({ translationKey, badgeText = undefined, icon, unit }
   return (
     <div className="flex flex-col pb-0.5">
       <div className="flex flex-row justify-between pt-4">
-        <div className="flex content-center gap-1.5">
+        <div className="flex content-center items-center gap-1.5">
           {icon && <div className="w-5">{icon}</div>}
-          <h3 className="text-md font-bold">{t(`${translationKey}.${timeAverage}`)}</h3>
+          <h2>{t(`${translationKey}.${timeAverage}`)}</h2>
```
Comment (madsnedergaard) [inline web/src/features/charts/ChartTitle.tsx:26]: What are the thoughts on making this a `h2`? In Figma it's using text-sm (at least in the file I looked at, sorry if you have another reference)
Screenshot 2024-07-01 at 15 03 56

**Review by madsnedergaard — COMMENTED** [review]
Maybe I'm just old and scared of changes (😅), but I feel that the text-size increases in various places are quite dramatic? 


_graph title_
Screenshot 2024-07-01 at 14 59 43

_toggle labels_
Screenshot 2024-07-01 at 14 59 24

File: web/src/features/charts/ChartTitle.tsx (hunk 1 above)
Comment (Alportan) [reply web/src/features/charts/ChartTitle.tsx:26]: Sorry, Figma is all over the place. The reference is [in the design system file for now](https://www.figma.com/design/BhpQbMBpicEYZeQ9aKKW7j/%E2%9A%A1%EF%B8%8F-The-Grid---Design-System-%E2%9A%A1%EF%B8%8F?node-id=959-12162&t=zREq1v94EsHwEdsT-1). It is an H2 there 🥳

**Comment by Alportan** [comment]
> Maybe I'm just old and scared of changes (😅), but I feel that the text-size increases in various places are quite dramatic?
> 

Haha, that makes sense. This is just the first step, but generally it is good to have a bigger text size. I do see why you mention this as the text-size does increase dramatically in quite a few places 😅 After this PR is merged, we'll be fine-tuning a few more components, and Viktor has already started the work with the historical picker and ranking section 🙌 

We do aim to increase the accessibility of our app, and there will be more things we'll tune, but we do need this global change to enable other changes 🙌

**Comment by madsnedergaard** [comment]
> > Maybe I'm just old and scared of changes (😅), but I feel that the text-size increases in various places are quite dramatic?
> 
> Haha, that makes sense. This is just the first step, but generally it is good to have a bigger text size. I do see why you mention this as the text-size does increase dramatically in quite a few places 😅 After this PR is merged, we'll be fine-tuning a few more components, and Viktor has already started the work with the historical picker and ranking section 🙌
> 
> We do aim to increase the accessibility of our app, and there will be more things we'll tune, but we do need this global change to enable other changes 🙌

Fair point, I'll fold to the argument of accessibility 🙇

**Review by madsnedergaard — APPROVED** [review]

**Comment by VIKTORVAV99** [comment]
Really nice to see this merged! Good work Silke!

**Pull request merged**